paragon/tokens/components/Button.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Defines overrides for the primary button variant.

[note] it doesn't look like all button variants are currently in the `alpha` release branch of the core Paragon tokens, so this addition is actually more of an "extension" than an override.